tests/test_minkowski_space.py

```diff
@@ -49,18 +49,18 @@ def test_inner_product_vectorization(self):
         self.assertTrue(gs.allclose(result, expected))
 
         result = self.metric.inner_product(n_points_a, one_point_b)
-        self.assertTrue(gs.allclose(result.shape, (n_samples, 1)))
+        self.assertTrue(gs.allclose(result.shape, (n_samples,)))
```

Comment:
I don't think this should be done through allclose no? We're comparing integers. Here and pretty much everywhere else?